packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,74 @@
+# Toolbar
+
+## Component Description
+
+A toolbar is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+When a set of controls is visually presented as a group, the toolbar role can be used to communicate the presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be an effective way of reducing the number of tab stops in the keyboard interface.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Storybook](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "static" | "contextual"
+- @attr size: "small" "medium" "large" | "medium"
+- @attr vertical: boolean | false
+  -- FAST uses `orientation`: enum { horizontal, vertical }
+
+### Outputs
+
+### Slots
+
+- start: Content which appears at the start of the toolbar, before the default slotted items.
+- default: Content which appears in the default location of the toolbar, left-aligned.
+- end: Content which appears at the end of the toolbar, right aligned.
+
+### CSS Variables
+
+None
+
+## Interactions
+
+None - Interactions such as hover and click are subject to the UX rules of the individual components slotted in the toolbar.
```

Comment:
This isn't entirely accurate for the FAST component. A toolbar has specific implementation details with regard to interaction model and the toolbar implements the ARIA practices guidance for that. That all will be inherited, but for instance, radio groups behave slightly differently within the toolbar, etc.

Reply:
Can we slot in other web components to serve as the "content" of the toolbar?

Reply:
That is the current API. Here's the current Fluent (non-Fluent 2) version which uses the same underlying Foundation component: https://fluent-components.azurewebsites.net/?path=/docs/components-toolbar--toolbar